packages/npm-lit/src/index.ts

```diff
@@ -0,0 +1,102 @@
+import type {Ctx, CtxSpy, Atom, Unsubscribe} from '@reatom/core'
+import {atom, throwReatomError} from '@reatom/core'
+import {nonNullable} from '@reatom/utils'
+import {LitElement, PropertyValues} from 'lit'
+
+
+type Constructor<T> = new (...args: any[]) => T;
+
+const compare = (a: unknown[], b: unknown[]) => {
```

Comment:
use `isShallowEqual` or short this

```suggestion
const isShallowEqual = (a: unknown[], b: unknown[]) => a.length === b.length && a.every((el, i) => el === b[i])
```

Reply:
ok, but: 
![изображение](https://user-images.githubusercontent.com/11953142/212738497-788e8b23-cc0d-472a-885e-5a3ea4b7bbef.png)

Reply:
Trust me, this is **nothing** :)